### PR TITLE
Fixes #30078 Add default_hostgroup plugin for the Katello scenario

### PR DESCRIPTION
**Pull request by apatelKmd**
Based on the recommendation provided in the support communication
https://community.theforeman.org/t/default-host-group-plugin-config-file-through-the-installer/18797/2

### CODE REVIEW

File: bin/create-migration [hunk 1]
```diff
@@ -16,8 +16,8 @@ directories.each do |directory|
     exit 2
   end
 
-  if File.basename(directory) == 'foreman.migrations'
-    # Used by the foreman scenario
+  if ['foreman.migrations', 'katello.migrations'].include?(File.basename(directory))
```
Comment (ehelms) [inline bin/create-migration]: I don't believe this change is needed. Katello and Foreman scenarios currently use different timestamp formats for their migrations and hence why this code exists as it does (see the else block).

**Comment by ehelms** [comment]
Could you please open a Redmine issue and attach it in the commit message?

File: bin/create-migration (hunk 1 above)
Comment (apatelKmd) [reply bin/create-migration]: Remvoed

**Comment by apatelKmd** [comment]
I have added the Redmine ticket number in the title https://projects.theforeman.org/issues/30078

**Comment by ehelms** [comment]
@apatelKmd Almost there! Can you edit the commit message to be the same as the title? That way when it goes into the git log it will have the right info to link it back to the Redmine issue.

**Comment by apatelKmd** [comment]
@ehelms Updated the commit message.

**Comment by ehelms** [comment]
@wbclark added some new tests here, could you help @apatelKmd with what changes are needed with this PR and the new tests?

**Comment by ekohl** [comment]
This whole change is not needed since it's already present:
https://github.com/theforeman/foreman-installer/blob/2cfd0523d41a85c024a7db17886b23c2bacfd2be/config/foreman-answers.yaml#L33
https://github.com/theforeman/foreman-installer/blob/2cfd0523d41a85c024a7db17886b23c2bacfd2be/config/katello-answers.yaml#L50

Looks like this was done prior to the installer merge so somewhere before 1.21 and my instructions on Discourse were incorrect.

https://github.com/theforeman/puppet-foreman/blob/master/manifests/plugin/default_hostgroup.pp would need to be modified to manage the config file. https://github.com/theforeman/puppet-foreman/pull/601 is a similar PR that could serve as inspiration.

My apologies for misleading here.

**Pull request closed without merging**